hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/AbfsHttpConstants.java

```diff
@@ -165,5 +169,20 @@ public static ApiVersion getCurrentVersion() {
    */
   public static final Integer HTTP_STATUS_CATEGORY_QUOTIENT = 100;
 
+  private static final String CPK_CONFIG_LIST =
```

Comment:
again, javadoc with + {@value}. for the IDEs, more than the docs themselves

Reply:
Agreed, Taken.